### ISSUE TITLE
Catch erroneous substitutions of hyphen-minus with en dash

### ISSUE THREAD

**Issue by civier** [MRtrix3/mrtrix3#2332]
Right now, when a user erroneously use an en dash instead of the hyphen-minus (often when copying commands from other sources such as webpages), e.g.:
```
mrinfo –info image.nii.gz
```
The error message will be:
```
mrinfo: [ERROR] unknown format for image "–info"
mrinfo: [ERROR] error opening image "–info"
```
This can be extremely confusing to novice and advanced users alike (many terminals, e.g., Mac default's one, display hyphen-minus and en dash exactly the same).
I suggest to catch this, and display an appropriate error message, e.g.
```
mrinfo: [ERROR] en dash is probably used instead of hyphen-minus in "–info". If you try to specify a path that starts with en dash, please be aware that mrtrix3 does not allow this.
```
In the same opportunity, we should make files that start with en dash prohibited (the same way that files that start with hyphen-minus are), if this is not already the case. 

The behaviour was confirmed when installing mrtrix3 using conda on 15/6/2021.

**Comment by jdtournier** [comment]
Check out #2193 - that should sort out the problem, and probably goes quite a bit beyond what you're suggesting here. Not merged yet, but if there's demand we can try to prioritise it...

**Comment by jdtournier** [comment]
closed with #2193

**Comment by civier** [comment]
Great. How do I give a thumbs-up in GitHub? :-)
